src/topology/opens.lean

```diff
@@ -164,3 +162,17 @@ end
 end opens
 
 end topological_space
+
+namespace continuous
+open topological_space
+
+def comap {f : α → β} (hf : continuous f) (V : opens β) : opens α :=
+⟨f ⁻¹' V.1, hf V.1 V.2⟩
+
+@[simp] lemma comap_id (U : opens α) : (continuous_id).comap U = U := by { ext, refl }
+
+@[simp] lemma comap_mono {f : α → β} (hf : continuous f) {V W : opens β} (hVW : V ⊆ W) :
```

Comment:
Is the simp attribute useful here?

Reply:
Hmmm, that's a mistake. Thanks for catching it.